core/src/block_processor.rs

```diff
@@ -48,9 +47,9 @@ pub struct TransactionInfo {
     pub signature: String,
     pub err: Option<TransactionError>,
     pub status: Result<(), TransactionError>,
-    pub cu_requested: Option<i64>,
-    pub prioritization_fees: Option<i64>,
-    pub cu_consumed: Option<i64>,
+    pub cu_requested: Option<u32>,
```

Comment:
nit:solana uses u64 for cu

Reply:
apparently CU requested in returned in u32.